stable/rocketchat/templates/ingress.yaml

```diff
@@ -18,7 +18,11 @@ spec:
   tls:
   - hosts:
     - {{ .Values.host }}
-    secretName: {{ template "rocketchat.fullname" . }}-tls
+{{- if (eq "-" .Values.ingress.secretName) }}
+    secretName: ""
+{{- else }}
+    secretName: {{ .Values.ingress.secretName | default (printf "%s-tls" (include "rocketchat.fullname" .)) }}
+{{- end }}
```

Comment:
There's a case where TLS will be enabled without a secret. Not sure that's a good idea.

Could we get something that looks more like the example chart (what `helm create` generates)?

Reply:
This is exactly what I need. I use a [default certificate](https://github.com/kubernetes/ingress-nginx/blob/master/docs/user-guide/default-ssl-certificate.md).

Reply:
Also I can change to what ```helm create``` generates but it will not be backward-compatible. Is it OK?

Reply:
@sathieu since the `secretName` is optional, would it be better to use `ne` instead of `eq` in the if, move the content of the else to the if, and get rid of the else? When the `secretName` is `-` isn't just not included in the `tls` section.

Reply:
Done. Thanks for the hint.